java-spiffe-core/build.gradle

```diff
@@ -0,0 +1,64 @@
+
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+
+    ext.protobufPluginVersion = '0.8.12'
+
+    dependencies {
+        classpath group: 'com.google.protobuf', name: 'protobuf-gradle-plugin', version: "${protobufPluginVersion}"
+    }
+}
+
+ext {
+    grpcVersion = '1.30.0'
+    nettyVersion = '4.1.50.Final'
+    protobufProtocVersion = '3.12.0'
+}
+
+apply plugin: 'com.google.protobuf'
+
+sourceSets {
+    main {
+        java {
+            srcDirs 'build/generated/source/proto/main/grpc'
+            srcDirs 'build/generated/source/proto/main/java'
+        }
+    }
+}
+
+protobuf {
+    protoc {
+        artifact = "com.google.protobuf:protoc:${protobufProtocVersion}"
+    }
+    plugins {
+        grpc {
+            artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}"
+        }
+    }
+    generateProtoTasks {
+        all()*.plugins {
+            grpc {}
+        }
+    }
+}
+
+dependencies {
+    implementation group: 'io.grpc', name: 'grpc-netty', version: "${grpcVersion}"
```

Comment:
`grpc-java` documentation recommends users to use `grpc-netty-shaded`. Is this library compatible with applications that already have a dependency on `grpc-netty-shaded`?

Reply:
I'm not sure about this, I'm investigating. So far I saw that we couldn't support `MAC_OC` through the `io.netty.channel.kqueue.KQueueEventLoopGroup` using the `grpc-netty-shaded` .

Reply:
I've testing using the shaded dependency, it works, so I replaced the netty and grpc dependencies by the grpc-netty-shaded. As KQueue events aren’t supported by the shaded version, the library will no longer support MAC_OS natively. This will be addressed in a separate issue to add some tooling/documentation to test the library in a MAC_OS system.